controller/notify.go

```diff
@@ -1,18 +1,28 @@
 package controller
 
 import (
+	"context"
 	"fmt"
 
 	"github.com/fabric8-services/fabric8-notification/app"
 	"github.com/fabric8-services/fabric8-notification/collector"
 	"github.com/fabric8-services/fabric8-notification/email"
 	"github.com/fabric8-services/fabric8-notification/jsonapi"
 	"github.com/fabric8-services/fabric8-notification/template"
+	"github.com/fabric8-services/fabric8-notification/token"
 	"github.com/fabric8-services/fabric8-wit/errors"
 	"github.com/fabric8-services/fabric8-wit/log"
 	"github.com/goadesign/goa"
 )
 
+const (
+	CVENotification string = "analytics.notify.cve"
+)
+
+var nTypeToSAMap = map[string]string{
+	CVENotification: "fabric8-gemini-server",
+}
+
```

Comment:
In future, if we want to add restriction for other notification_type then just add entry here.  Currently, only one service_name is possible but we can extend it to service_name_array easily.